### ISSUE TITLE
ignited daemon mode crash due to empty status runtime

### ISSUE THREAD

**Issue by davxiao** [weaveworks/ignite#867]
I got the error when following the [guide](https://ignite.readthedocs.io/en/stable/installation/) under the section `Verify the micro-VM is accessible (username: root, password: root)`

Version information as follows:

Ubuntu 20.04.3 LTS
ignite and ignited installed latest version 

```
$ ignite version
Ignite version: version.Info{Major:"0", Minor:"10", GitVersion:"v0.10.0", GitCommit:"4540abeb9ba6daba32a72ef2b799095c71ebacb0", GitTreeState:"clean", BuildDate:"2021-07-19T20:52:59Z", GoVersion:"go1.16.3", Compiler:"gc", Platform:"linux/amd64", SandboxImage:version.Image{Name:"weaveworks/ignite", Tag:"v0.10.0", Delimeter:":"}, KernelImage:version.Image{Name:"weaveworks/ignite-kernel", Tag:"5.10.51", Delimeter:":"}}
Firecracker version: v0.22.4
```

```
ubuntu 31|22:51 [bin] docker version
Client:
 Version:           20.10.7
 API version:       1.41
 Go version:        go1.13.8
 Git commit:        20.10.7-0ubuntu1~20.04.1
 Built:             Wed Aug  4 22:52:25 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server:
 Engine:
  Version:          20.10.7
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.13.8
  Git commit:       20.10.7-0ubuntu1~20.04.1
  Built:            Wed Aug  4 19:07:47 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.5.2-0ubuntu1~20.04.2
  GitCommit:
 runc:
  Version:          1.0.0~rc95-0ubuntu1~20.04.2
  GitCommit:
 docker-init:
  Version:          0.19.0
  GitCommit:
```

**Comment by darkowlzz** [comment]
Hi @davxiao , thanks for reporting this issue. ignited isn't well tested at the moment due to which we weren't aware of this issue. I was able to reproduce it and created a fix for it in #868 . Also added an e2e test so that we get to know the next time it breaks due to some API changes.
If you could try the patch in #868 and provide any feedback, that would be very helpful in making ignited better.

Thanks.

**Comment by stealthybox** [comment]
Thank you for the detailed report :+1:
